src/Module/Photo.php

```diff
@@ -82,24 +82,28 @@ public static function rawContent(array $parameters = [])
 				$square_resize = !in_array($parameters['type'], ['media', 'preview']);
 			}
 
-			if (!empty($parameters['nickname_ext'])) {
-				if (in_array($parameters['type'], ['contact', 'header'])) {
-					$guid = pathinfo($parameters['nickname_ext'], PATHINFO_FILENAME);
-					$account = DBA::selectFirst('account-user-view', ['id'], ['guid' => $guid], ['order' => ['uid' => true]]);
-					if (empty($account)) {
-						throw new HTTPException\NotFoundException();
-					}
+			if (!empty($parameters['guid'])) {
+				$guid = pathinfo($parameters['guid'], PATHINFO_FILENAME);
```

Comment:
We aren't expecting a file extension in this parameter.
```suggestion
				$guid = $parameters['guid'];
```